### DISCUSSION TITLE
particle system

### DISCUSSION THREAD

**Discussion by Trusiak** [enable3d/enable3d#327] [General]
enable3d to its full happiness lacks a real particle system. 
Of course, we can theoretically use external systems native to THREE.JS, but by changing the structure of enable3d,
it is impossible to implement such systems (mainly it comes down to the prohibition of calling requestPerFrame, and the duplication of renderers when installing external libraries). 

if we do not want to create a new particle system, we should at least adapt the current e.g. three NEBULA to the enable engine.

**Comment by yandeu** [comment]
I have never used a particle system for enable3d/three.js.
I will immediately have a look at enable3d + nebula.

**Reply by yandeu** [reply]
Done.

I tried to implement [this example](https://codesandbox.io/s/three-nebula-quickstart-kz6uv). Works without any issues. Very easy to implement.

See [this screenshot](https://user-images.githubusercontent.com/20306025/191850910-a4932bc7-4e49-4de2-8790-707fffe187a5.png).

---

Feel free to add this and/or other particle examples to the website: https://github.com/enable3d/enable3d-website
